Makefile

```diff
@@ -183,7 +183,7 @@ pex: writeversion
 	ls dist/*.whl | while read whlfile; do pex $$whlfile --disable-cache -o dist/kolibri-`cat kolibri/VERSION | sed 's/+/_/g'`.pex -m kolibri --python-shebang=/usr/bin/python; done
 
 i18n-extract-backend:
-	python -m kolibri manage makemessages -- -l en --ignore 'node_modules/*' --ignore 'kolibri/dist/*'
+	python -m kolibri manage makemessages -- -l en --ignore 'node_modules/*' --ignore 'kolibri/dist/*' --ignore 'venv/*'
```

Comment:
is this general, or dev environment specific? (for example, my pipenv virtual environment is in `.venv`)

Reply:
Considering - `--include` (or equivalent) or `--ignore=*/site-packages/*`

Reply:
Could make it simpler by adding a `cd` into the kolibri dir, and then you don't need to ignore these top level folders.

Can also specify `--extension=html,py`, but by default it should ignore JS files anyway.